docs/docs/install/environment-variables.md

```diff
@@ -180,17 +177,16 @@ Typesense URL example JSON before encoding:
 
 | Variable                                    | Description                    |        Default        | Services         |
 | :------------------------------------------ | :----------------------------- | :-------------------: | :--------------- |
-| `MACHINE_LEARNING_MIN_FACE_SCORE`           | Minimum Face Score             |         `0.7`         | machine learning |
 | `MACHINE_LEARNING_MODEL_TTL`                | Model TTL                      |         `300`         | machine learning |
-| `MACHINE_LEARNING_EAGER_STARTUP`            | Eager Startup                  |        `true`         | machine learning |
-| `MACHINE_LEARNING_MIN_TAG_SCORE`            | Minimum Tag Score              |         `0.9`         | machine learning |
-| `MACHINE_LEARNING_FACIAL_RECOGNITION_MODEL` | Facial Recognition Model       |      `buffalo_l`      | machine learning |
-| `MACHINE_LEARNING_CLIP_TEXT_MODEL`          | Clip Text Model                |    `clip-ViT-B-32`    | machine learning |
-| `MACHINE_LEARNING_CLIP_IMAGE_MODEL`         | Clip Image Model               |    `clip-ViT-B-32`    | machine learning |
-| `MACHINE_LEARNING_CLASSIFICATION_MODEL`     | Classification Model           | `microsoft/resnet-50` | machine learning |
 | `MACHINE_LEARNING_CACHE_FOLDER`             | ML Cache Location              |       `/cache`        | machine learning |
 | `TRANSFORMERS_CACHE`                        | ML Transformers Cache Location |       `/cache`        | machine learning |
```

Comment:
`TRANSFORMERS_CACHE` can be removed. It doesn't actually do anything since the cache is set explicitly in the code to the model's own cache folder.

Reply:
Also, there are a few new threading env variables that are missing here: `MACHINE_LEARNING_REQUEST_THREADS`, `MACHINE_LEARNING_MODEL_INTER_OP_THREADS`  and `MACHINE_LEARNING_MODEL_INTRA_OP_THREADS`. 

`MACHINE_LEARNING_REQUEST_THREADS`: sets the size of the request thread pool, defaults to cpu count

`MACHINE_LEARNING_MODEL_INTER_OP_THREADS`: sets the number of model ops to process in parallel, defaults to 1

`MACHINE_LEARNING_MODEL_INTRA_OP_THREADS`: sets the number of threads to handle each model op, defaults to 1
* Recommended to try changing this first rather than `MACHINE_LEARNING_MODEL_INTER_OP_THREADS` if making changes

Reply:
I don't think I understand exactly the function of INTER_OP_THREADS vs INTRA_OP_THREADS - well, I understand the cardinality effects on the concurrency, but I don't understand their purpose so I might not have explained them well. Suggestions are very welcome :-)